seong-geun/app.js

```diff
@@ -0,0 +1,46 @@
+const http = require("http");
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+
+const dotenv = require("dotenv");
+dotenv.config();
+
+const { DataSource } = require("typeorm");
+
+const appDataSource = new DataSource({
+  type: process.env.DB_CONNECTION,
+  host: process.env.DB_HOST,
+  port: process.env.DB_PORT,
+  username: process.env.DB_USERNAME,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_DATABASE,
+});
+
+appDataSource
+  .initialize()
+  .then(() => {
+    console.log("Data Sourece has been initialized!");
+  })
+  .catch((error) => {
+    console.log("Alert Error. Need modify!", error);
+  });
+
+const app = express();
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan("dev"));
+
+app.get("/ping", cors(), function (req, res, next) {
+  res.json({ message: "pong" });
+});
+
+const server = http.createServer(app);
+const PORT = process.env.PORT;
+
+const start = async () => {
+  server.listen(PORT, () => console.log(`server is listening on ${PORT}`));
```

Comment:
```suggestion
  app.listen(PORT, () => console.log(`server is listening on ${PORT}`));
```
http 모듈을 활용한 server라는 변수 대신, express를 활용하여 위에서 선언한 app이라는 실행시키도록 작성해주세요~